src/adapters/JSONFile.ts

```diff
@@ -1,5 +1,5 @@
 import * as fs from 'fs'
-import * as mutexify from 'mutexify'
+import mutexify = require('mutexify')
```

Comment:
typo?

Reply:
Actually no, this is the recommended way to do this. I was researching how to write type definitions in a good way when I came across this:
 
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require
> When exporting a module using export =, TypeScript-specific import module = require("module") must be used to import the module.

I read more about it here: https://stackoverflow.com/a/39415662

It's possible to fake it using a namespace merged with the function declaration but this is not encouraged by DefinitelyTyped amongst others: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports